decidim-admin/app/views/decidim/admin/organization_privacy_policy/edit.html.erb

```diff
@@ -0,0 +1,43 @@
+<div class="grid-container full">
```

Comment:
I see that this is copy and pasted from other places (like https://github.com/decidim/decidim/blob/b2aac44b358c16d1737e2488844c8a97ea04de4a/decidim-admin/app/views/decidim/admin/shared/landing_page/edit.html.erb and https://github.com/decidim/decidim/blob/e06bef740022aa97e5d50faf8847130dee12445c/decidim-admin/app/views/decidim/admin/organization_homepage/edit.html.erb). I was thinking about if we should refactor this to remove the duplication (like moving it to a partial or similar). But then I remembered that we talked about changing this to another UI that didn't use Content Blocks drag and drop. Is that still the plan?

Reply:
Yes, there will be a slight different UI, because with the following PR (ToS content_block sections) multiple content_block of the same type will be possible. If possible I'll try to DRY up.